content/content-strategy/define-user-content-needs/create-actionable-artefacts/user-profiles/section2.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: text/textblock
+---
+
+### [2]A user characteristics gauge (in relation to the product or service)
+Depending on the nature of the project you are working on, you may like to include a characteristics gauge. This will allow you to easily see the differences between each of your different profiles. The gauge can be a simple line that rates a participant from low to high. You can add the characteristics depending on what is relevant to your project. Here are a few examples.
+  * Knowledge or expertise
+  * Level of interest
+  * Context of interest
+  * Breadth of usage 
+  * Experience in related task
+  * Digital literacy/format preference 
+
+Build on your profile by answering questions about their needs, pain points, risks and motivation.
+
+**Needs / Tasks**
```

Comment:
Suggestion: if these words are used to label sub sections, please use a heading 4 instead of bolded text. Helps users and robots understand the page's composition :)

`####` before 'Needs / Tasks' etc.

Reply:
Fixed